test/chi/chi-vue/data-table-portal.pug

```diff
@@ -42,7 +42,7 @@ script.
           bordered: false,
           hover: false,
           size: "md",
-          striped: false,
+          striped: true,
```

Comment:
Same here. Is this change expected?

Reply:
Yes, UX/UI requested making the striping optional.
Previously enabling `portal: true`, striping was automatic.
This is a test page .pug file. The test was failing as now the striping is optional ;)